conda_forge_tick/update_recipe/version.py

```diff
@@ -557,20 +575,28 @@ def update_version_feedstock_dir(
         )
 
 
-def _update_version_feedstock_dir_local(feedstock_dir, version, hash_type):
-    with open(os.path.join(feedstock_dir, "recipe", "meta.yaml")) as f:
+def _update_version_feedstock_dir_local(
+    feedstock_dir, version, hash_type
+) -> (bool, set):
+    if os.path.join(feedstock_dir, "recipe", "recipe.yaml"):
+        return update_version_v1(feedstock_dir, version, hash_type)
```

Comment:
`update_version_v1` returns the updated recipe (not a bool), it doesn't write it back to recipe.yaml. That's handled below in this funciton, but because of this `return`, the write-back never happens, so running `pixi run bump` results in no changes written to the file.

Reply:
fixed, thanks!